src/proxy/routes/index.js

```diff
@@ -1,17 +1,21 @@
-/* eslint-disable max-len */
 const express = require('express');
 const proxy = require('express-http-proxy');
-// eslint-disable-next-line new-cap
-const router = express.Router();
+const router = new express.Router();
 const chain = require('../chain');
+const config = require('../../config');
+
+if (config.getAllowSelfSignedCert()) {
+  process.env['NODE_TLS_REJECT_UNAUTHORIZED'] = 0;
+}
 
 router.use(
   '/',
-  proxy('https://github.com', {
+  proxy(config.getProxyUrl(), {
```

Comment:
I appreciate this being configurable, works well if you had to support flows to GitLab and other Git providers :+1: 

Maybe a separate feature but would this configuration ever be an array of URLs that all can be proxied by one git-proxy server simultaneously? The Express routes can be created as an array of URLs to pass to the proxy middleware.

The alternative approach can be to deploy separate servers with their own configured URLs, which could be a fair trade-off.

Reply:
@coopernetes - yes. I'd suggest we open this up as another issue 👍